<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WInvitePageClass</class>
 <widget class="QWidget" name="WInvitePageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WInvitePage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 12px &quot;微软雅黑&quot;;
}

QLabel#lb_peo{
	background-image: url(:/QtTest/icon/Setting/peo.png);
}

QPushButton#pb_search{
	border:none;
	background-image: url(:/QtTest/icon/Setting/search.png);
}

QLineEdit{
	border:none;
	background-color: rgba(255, 255, 255,0);
}

QFrame#frame_1,#frame_2{
	border:1px solid #c9c9c9;
	border-radius:8px;
	background-color: rgb(255, 255, 255);
}

QPushButton#pb_invite{
	font: 16px &quot;微软雅黑&quot;;
	border:none;
	border-radius:6px;
	color: rgb(255, 255, 255);
	background-position: center;
    background-repeat: no-repeat;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 #0066FF, stop:1 #BD00FF);
}

QPushButton#pb_pre{
	border:none;
	background-position: center;
    background-repeat: no-repeat;
	background-image: url(:/QtTest/icon/Setting/pre.png);
}

QPushButton#pb_next{
	border:none;
	background-position: center;
    background-repeat: no-repeat;
	background-image: url(:/QtTest/icon/Setting/next.png);
}

QComboBox{
	border:1px solid #E0E7ED;
	border-radius:4px;
}
QComboBox::down-arrow {
	image: url(:/QtTest/icon/Speech/down.png);
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QTableView{
	border: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgb(0,102,255), stop:1 rgb(189,0,255));
	border-radius: 8px;
	background-color:transparent;
	font-family: &quot;Microsoft YaHei UI&quot;;
	font-size: 14px;
	color: #333333;
	text-align: left;
}

QHeaderView{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0,102,255,0.10), stop:1 rgba(189,0,255,0.10));
	border:none;
	border-top-left-radius: 8px;
	border-top-right-radius: 8px;

}

QHeaderView::section{
	border-radius: 8px 8px 0 0;
    border: none;
    height:44px;    /*表头高度*/
    color: #333333;
	font-family: &quot;Microsoft YaHei UI&quot;;
	font-weight: Bold;
	font-size: 14px;
	background-color:transparent;
    text-align: left;
}

QTableView::item{ 
    background-color: white;
    border: none;
    background-color:rgba(0,0,0,0.04); /*交替行，第二行*/
    /*alternate-background-color*/
	selection-background-color: white;
    selection-color: black;
}
QTableView::item:!alternate:!selected{
    background-color: transparent;    
    selection-background-color: rgb(220, 220, 220);  
    selection-color: black;
}


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>19</height>
      </size>
     </property>
     <property name="text">
      <string>Invite users to your team by putting there usernames in below field</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QFrame" name="frame_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>45</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_peo">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_invite">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter user name to add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_invite">
       <property name="minimumSize">
        <size>
         <width>182</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>182</width>
         <height>45</height>
        </size>
       </property>
       <property name="text">
        <string>Invite member</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 18px &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Member</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>213124</width>
         <height>45</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="le_search">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Search any record</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_search">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>410</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>410</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="WPageCtlWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WPageCtlWidget</class>
   <extends>QWidget</extends>
   <header>model/WPageCtlWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../AiSound.qrc"/>
 </resources>
 <connections/>
</ui>
